redis_shard.go

```diff
@@ -39,14 +39,17 @@ const (
 	redisDataBatchLimit = 64
 )
 
-type redisConnPool interface {
-	Get() redis.Conn
-	Close() error
+type redisLogger struct{}
+
+func (redisLogger) Printf(ctx context.Context, format string, v ...interface{}) {}
+
+func init() {
+	redis.SetLogger(redisLogger{})
```

Comment:
This seems that we disable logger globally here? So that this can affect apps which import centrifuge library and use `go-redis`. What happens if we won't call `SetLogger` at all - does it log a lot into stdout or what happens?

Global logger is such an unfortunate thing for a library which may be used from different parts of final application..

Reply:
> this can affect apps which import centrifuge library and use go-redis
Ah, you're right, we can't disable the global logger. 

> does it log a lot into stdout or what happens?
It logs into stderr.